File: Data Structures & Algorithms/top-k-elements-in-list/submission-3.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        
        hashMap=defaultdict(int)

        for num in nums:
            hashMap[num]+=1

        sorted_dict = dict(sorted(hashMap.items(),key=lambda item:item[1],reverse=True))
        l=list(sorted_dict.keys())
        return l[0:k]

File: Data Structures & Algorithms/climbing-stairs/submission-3.java
class Solution {
    public int climbStairs(int n) {
        int [] nums = new int [n];
        nums[0]=1;
        if(n>1) nums[1]=2;
        for(int i=2;i<n;i++)
        {
            nums[i]+=nums[i-1]+nums[i-2];
        }
        return nums[n-1];
    }
}
